### PR TITLE
Helm Chart

### PR DESCRIPTION
**Pull request by dberuben**
I created a helm chart for my own deployment.i would love to share with people. It can be improved, very basic helm chart. :D

### CODE REVIEW

**Comment by angelbarrera92** [comment]
Ciao @dberuben! We will take a look asap 👍

**Comment by loeken** [comment]
thank you @dberuben just tested the chart from your fork and it worked like a charm! thumbs up for getting this in here!

File: helm_chart/templates/ClusterRoleBinding.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: {{ include "permission-manager.serviceAccountName" . }}
+roleRef:
+  kind: ClusterRole
+  name: {{ include "permission-manager.serviceAccountName" . }}
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "permission-manager.serviceAccountName" . }}
+    namespace: {{ .Values.config.namespace }}
```
Comment (phisco) [inline helm_chart/templates/ClusterRoleBinding.yaml:12]: any particular reason to pass it like this and not through `.Release.Namespace` (see [here](https://helm.sh/docs/chart_template_guide/builtin_objects/))

**Review by phisco — CHANGES_REQUESTED** [review]

File: helm_chart/templates/ClusterRoleBinding.yaml (hunk 1 above)
Comment (dberuben) [reply helm_chart/templates/ClusterRoleBinding.yaml:12]: No, let me fix that 👍

Comment (dberuben) [reply helm_chart/templates/ClusterRoleBinding.yaml]: Thank you @phisco it fixed, i also fixed the secret name which it was hardcoded

**Review by phisco — APPROVED** [review]
LGTM! 

Thanks @dberuben! 🎉  

@angelbarrera92 now we can add the github action to build and host the chart releases.

**Comment by angelbarrera92** [comment]
> LGTM!
> 
> Thanks @dberuben! 🎉
> 
> @angelbarrera92 now we can add the github action to build and host the chart releases.

Copying also @Rampage1xx :)

**Pull request merged**